packages/squiggle-lang/src/reducer/lambda.ts

```diff
@@ -193,6 +207,20 @@ export class BuiltinLambda extends BaseLambda {
     }
     throw new REOther(showNameMatchDefinitions());
   }
+
+  toCalculator(): Calculator {
+    const longestSignature = maxBy(this.signatures(), (s) => s.length) || [];
```

Comment:
This is a reasonable heuristic, but it could cause some frustration if the longest signature is not the most natural one.

Brainstorming other, more general approaches:
1) `Calculator.make(fn) -> Calculator[]` (return all possible versions)
2) `Calculator.make(fn.definitions[i]) -> Calculator`, where `fn.definitions[i]` returns a specialized function with a single signature (but this is awkward because it means that `fn.definitions[3][0][0][0][0][0]` should be allowed)
3) `Calculator.make(fn, 3) -> Calculator` (same as `Calculator.make(fn)[3]` in (1) above)

Reply:
Thanks for the ideas, but I don't think I'm a fan of those.
1. This could be messy, especially if there are many definitions (you'll get 3-10 on some items). I'd expect it to surprise users.
2. Agreed it's awkward. Slow to type out.
3. Also seems weird, especially as the order of definitions might be a bit chaotic.

One nice thing is that this issue should only come into play for builtin functions - users functions only have one definition.

My guess is that we should leave for now. However, here are some options for later:
1. Maybe, if there are multiple definitions, then if you try ``Calculator.make()`` without any definitions, it shows you an error, but gives you a list of definitions as strings. You then can plug them in as, ``Calculator.make(defString)``.

2. We output a Dict, where keys have to do with the definitions somehow.

3. We output a small dropdown menu of definitions, after which the correct form will appear.